cqf-fhir-utility/src/main/java/org/opencds/cqf/fhir/utility/repository/InMemoryFhirRepository.java

```diff
@@ -75,14 +79,13 @@ public <T extends IBaseResource, I extends IIdType> T read(
 
     @Override
     public <T extends IBaseResource> MethodOutcome create(T resource, Map<String, String> headers) {
```

Comment:
What is the reason for these changes?

Nitpick: use var when able

Reply:
I needed to add `theId` of the created/updated resource to the `MethodOutcome` so that `transactionStub` could add it to the return bundle